stories/legend.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+
+import BarChart from '../src/components/bar-chart/'
+import PieChart from '../src/components/pie-chart'
+
+import barChartData from './data/bar-chart-data'
+import lineChartData from './data/line-chart-data'
+import pieChartData from './data/pie-chart-data'
+import ResponsiveChartWrapper from './responsive-chart-wrapper'
+
+
+const title = 'My Title'
+
+storiesOf('Legend', module)
+  .add('Legend disabled', () => (
+    <ResponsiveChartWrapper>
+      <BarChart
+        data={ lineChartData }
+        indexBy='country'
+        keys={ ['amount'] }
+        axisBottomLegendLabel='Address City'
+        axisLeftLegendLabel='Visitors'
+        groupMode='grouped'
+        disableLegend={ true }
```

Comment:
oh, when handling `true/false` values, you don't usually need to do the equality `disableLegend={true}` but just the declaration of the attribute `disableLegend` already makes it true (since its default is false)